types/session_price.yaml

```diff
@@ -0,0 +1,73 @@
+description: Types to provide the session price
+types:
+  SessionCostChunk:
+    description: >-
+      A chunk of the session cost. The total cost of the session is the sum of
+      all chunks.
+    type: object
+    additionalProperties: false
+    properties:
+      timestamp_from:
```

Comment:
One thing - for our use case (night tariff), the minutes we charged infrastructure fee for is not simply `t_stop - t_end`
Can we add this information here? Like infrastructure cost or something like that